DevEdu.API/DevEdu.Business.Tests/StudentAnswerOnTaskServiceTests.cs

```diff
@@ -0,0 +1,209 @@
+﻿using DevEdu.Business.Services;
+using DevEdu.DAL.Repositories;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace DevEdu.Business.Tests
+{
+    public class StudentAnswerOnTaskServiceTests
+    {
+        private Mock<IStudentAnswerOnTaskRepository> _studentAnswerOnTaskRepoMock;
+
+        [SetUp]
+        public void Setup()
+        {
+            _studentAnswerOnTaskRepoMock = new Mock<IStudentAnswerOnTaskRepository>();
+        }
+
+
+        [Test]
+        public void AddStudentAnswerOnTask_ExistingTaskIdAndStudentIdAndStudentAnswerOnTaskInputModelPassed_StudentAnswerWasAdded()
+        {
+            // Given
+            var studentAnswerDto = StudentAnswerOnTaskData.GetStudentAnswerOnTaskDto();
+            int taskId = StudentAnswerOnTaskData.TaskId;
+            int userId = StudentAnswerOnTaskData.UserId;
```

Comment:
it's a bad practice to have such values as consts in static class
it's better to declare them inside test methods